Installer/Installer_Linux/install.sh

```
@@ -168,7 +168,7 @@ function install_dependencies()
         wget -qO- ${hyperg} > ${hyperg_pack}
         [[ -d $HOME/hyperg ]] && rm -rf $HOME/hyperg
         tar -xvf ${hyperg_pack} >/dev/null
-        mv hyperg $HOME/
+        mv hyperg $HOME/ 2>dev/null
```

1. it's not related to building taskcollector
2. maybe something like `[[ "$PWD" != "$HOME" ]] && mv hyperg $HOME/`?

---

1. No its not but its not worth PR for such a change.
2. Maybe good idea, but don't want to change behavior just before release.

---

But this might also swallow other error messages. I think we should leave as it was before, or do it properly.